### ISSUE TITLE
Background threads do not terminate when using native callback.

### ISSUE THREAD

**Issue by hanazuki** [hanazuki/node-jsonnet#18]
The following program must terminate:
```
const { Jsonnet } = require("@hanazuki/node-jsonnet");
const jsonnet = new Jsonnet();
jsonnet.nativeCallback("id", (x) => x, "x");
```

**Comment by hanazuki** [comment]
Bisect suggests bdd90a591104b01c6cb828440e53abaa354b65ca as the first bad commit

**Comment by hanazuki** [comment]
Fixed in a5ae1896dca8e3ca02195b590627bcfd89fe4caf
